test/fixtures/unix.js

```diff
@@ -4867,93 +4923,93 @@ export const flag = {
   "sample strings": [
     {
       input: "foobar",
-      expected: "foobar",
+      expected: { unquoted: "foobar", quoted: "'foobar'" },
```

Comment:
Quotes may differ from one shell to another (even on the same system), update the `flag` fixtures object to provide fixtures per shell (extends to `test/fixtures/win.js`).

Reply:
Resolved with 57c8be7b8f4dab9e2d087b9954c4e5b18e783597